python/ray/tune/tests/test_soft_imports.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import unittest
+
+
+class TestSoftImports(unittest.TestCase):
+    """Tests whether it's possible to use Ray Tune without soft dependencies
+    """
+
+    def testSoftImports(self):
```

Comment:
nice! can you also check that these libraries are not available? i.e., use importlib to check for the availability of libraries, just incase someone accidentally introduces a regression that downloads extra dependencies

Reply:
Should be good now!